udpa/annotations/status.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package udpa.annotations;
+
+import "google/protobuf/descriptor.proto";
+
+// Magic number in this file derived from top 28bit of SHA256 digest of
+// "udpa.annotation.status".
+extend google.protobuf.MessageOptions {
+  StatusAnnotation message_status = 222707719;
```

Comment:
What's the use case for options other than file? I don't think we want that?

Reply:
Good question; I was thinking we could have WiP messages, but yeah, let's not allow this too fine grained, or it's going to be too confusing.